src/Storage/Notmuch.hs

```diff
@@ -40,6 +40,9 @@ removeTags tgs = over tags (filter (`notElem` tgs))
 getTags :: (ManageTags a) => a -> [T.Text]
 getTags = view tags
 
+isNew :: (ManageTags a) => T.Text -> a -> Bool
+isNew ignoredTag x = ignoredTag `elem` (view tags x)
```

Comment:
`ignoredTag` seems like a strange choice for the arg name.  Even the name of the function
is a bit off - seems like ``hasTag t x = t `elem` view tags x`` would be a better names.

Reply:
Yeah I think I stuffed that one up. Sorry :(